### PR TITLE
Fixed a few env variable sets, expanding on demo simplifications done…

### PR DESCRIPTION
**Pull request by billpratt**
… in previous commit

Expanded on simplifications done in af078c83879830a3c9dc047c8c78b94722aff9bd

### CODE REVIEW

**Comment by kkmsft** [comment]
@billpratt - could you please rebase you branch so that we can get it merged.

File: docs/tutorial/README.md [hunk 1]
```diff
@@ -99,19 +97,28 @@ The demo is basic, but does prove the concept.
 
 ### 3.1. Create an Azure Id
 
-We will be assigning the demo pod an [Azure Managed Service Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview). The Azure Id will need to be in the same Resource Group *as was created automatically by the provisioning of the AKS cluster* [see this issue for more information](https://github.com/Azure/aad-pod-identity/issues/38).
+We will be assigning the demo pod an [Azure Managed Service Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview).
+
 
 You might find the Resource Group name with
 
 `az group list | grep $RG`
```
Comment (kkmsft) [inline docs/tutorial/README.md]: Any idea why this has to be done now ?

File: docs/tutorial/README.md [hunk 2]
```diff
@@ -99,19 +97,28 @@ The demo is basic, but does prove the concept.
...
 
-Then set the environment variable
+Then set the environment variables
+
+`export AZ_IDENTITY_NAME="az-identity-name"`
+
+`export MC_RG="resource-group-name"`
```
Comment (kkmsft) [inline docs/tutorial/README.md]: Is there a way we can distinguish this and the $RG grep we do above ?

File: docs/tutorial/scripts/3-deploy-demo/4-deploy-id-binding.sh [hunk 3]
```diff
@@ -2,12 +2,16 @@
...
-your binding name from step 3's AzureIdentity, and a pod selector to assign tge Id"
+binding_name="${AZ_IDENTITY_NAME}-binding"
+binding_selector="${binding_name}-selector"
 
-read -p "Press enter to continue"
+perl -pi -e "s/BINDING_NAME/${binding_name}/" ${file}
+perl -pi -e "s/AZURE_IDENTITY_NAME/${AZ_IDENTITY_NAME}/" ${file}
+perl -pi -e "s/SELECTOR_VALUE/${binding_selector}/" ${file}
 
 set -x
```
Comment (kkmsft) [inline docs/tutorial/scripts/3-deploy-demo/4-deploy-id-binding.sh]: Could we use the ${file} in kubectl apply as well ?

**Comment by aramase** [comment]
Closing this PR since it's been out here for a while. Please feel free to reopen the PR with updated changes.

**Pull request closed without merging**